CHANGELOG.rst

```diff
@@ -22,6 +22,7 @@ Changed
 
 Fixed
 ~~~~~~~~~
+* Added monkey patch fix to st2stream to enable it to work with mongodb via SSL.
```

Comment:
Please follow the Changelog format including the PR number and type of the change if it's (bug fix) (enhancement), etc. See other examples below.
You're also welcome to include your Name or github Username as a contributor credit.